File: 104 Maximum Depth of Binary Tree/104.c
int maxDepth(struct TreeNode* root){
    int x, y;
    if(root){
        x = maxDepth(root->left);
        y = maxDepth(root->right);
    }else{
        return 0;
    }
    return (x >= y) ? (x+1) : (y+1);
}
//Min:4ms

File: 104 Maximum Depth of Binary Tree/104.py
class Solution(object):
    def maxDepth(self, root):
        if (root == None):
            return 0
        x = self.maxDepth(root.left)
        y = self.maxDepth(root.right)
        if (x>=y):
            return (x+1)
        return (y+1)

#Min:60ms